src/components/TableCard.tsx

```diff
@@ -0,0 +1,102 @@
+import { ReactNode, useMemo } from "react";
+import { useHover } from "react-aria";
+import { Css, Palette } from "src/Css";
+import { ButtonMenu, MenuItem } from "./ButtonMenu";
+import { Tag, TagType } from "./Tag";
+
+export type TableCardType = "card" | "list";
+
+type CardTag = {
+  text: string;
+  type?: TagType;
+};
+
+export interface TableCardProps {
+  title: string;
+  subtitle: string;
+  metadata: ReactNode;
+  image: string;
+  // contain displays entire image, cover fills the space
+  imageFit?: "contain" | "cover";
+  type?: TableCardType;
+  bordered?: boolean;
+  disabled?: boolean;
+  buttonMenuItems?: MenuItem[];
+  tag?: CardTag;
+}
+
+export function TableCard(props: TableCardProps) {
+  const {
+    title,
+    subtitle,
+    metadata,
+    image,
+    imageFit = "contain",
+    type = "card",
+    bordered = false,
+    disabled: isDisabled = false,
+    buttonMenuItems,
+    tag,
+  } = props;
+  const { hoverProps, isHovered } = useHover({ isDisabled });
+  const isList = type === "list";
+  const imageSize = isList ? 96 : bordered ? 224 : 256;
+
+  const styles = useMemo(
+    () => ({
+      ...baseStyles(type),
+      ...(isList && listStyles),
+      ...(bordered && borderedStyles),
+      ...(isHovered && cardHoverStyles),
+      ...(isDisabled && disabledStyles),
+    }),
+    [isDisabled, isHovered, bordered, type, isList],
+  );
+
+  return (
+    <div css={styles} {...hoverProps}>
+      {/* Image */}
+      <div css={Css.wPx(imageSize).hPx(imageSize).ba.br8.bcGray300.overflow("hidden").df.asc.jsc.relative.$}>
+        {/* overlay on hover */}
+        <div
+          css={{
+            ...Css.absolute.top0.left0.w100.h100.add("opacity", 0).$,
+            ...(isHovered && !isList && imageHoverStyles),
+          }}
+        />
```

Comment:
thought to take or leave: what you have is 👍 but it might be cool to explore if a CSS filter would do the trick here to give us fine grain control over the behavior and skip dealing with an absolute positioned div, etc. https://css-tricks.com/almanac/properties/f/filter/
Been around for a while now so have very good browser support finally

Reply:
thank you for suggestion! I was able to apply a brightness filter instead